src/r/DESCRIPTION

```diff
@@ -0,0 +1,26 @@
+Package: psychrolib
+Type: Package
+Title: Psychrometric Properties of Moist and Dry Air
+Version: 2.3.0
+Authors@R: c(person("Hongyuan", "Jia", email = "hongyuan.jia@bears-berkeley.sg", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-0075-8183")),
+             person("Jason", "Banfelder", email = "jason@banfelder.net", role = "aut"),
+             person("D.", "Meyer", role = "aut"),
+             person("D.", "Thevenard", role = "aut"))
+Description:
```

Comment:
Can we please change this to:
```
PsychroLib is a library of functions to enable the calculation of psychrometric properties of moist and dry air. Versions of PsychroLib are available for Python, C, C#, Fortran, R, JavaScript, Microsoft Excel Visual Basic for Applications (VBA). The library works in both metric (SI) and imperial (IP) systems of units. For more information about PsychroLib, or to report bugs and get information on how to cite it, please refer to the [official repository on GitHub](https://github.com/psychrometrics/psychrolib).
```
This reflect what we have in the main README already.

Reply:
I am afraid here we have to stick to CRAN's conventions

According to my experience, CRAN maintainer will not allow to mention the package itself in `DESCRIPTION`. All external programs or libraries should be single quoted with links surrounded by `<>`. That's why I chose the current words in `DESCRIPTION`.

In terms of the repo link, there are already fields `URL` and `BugReports` to reflect.

In terms of citing psychrolib, I will upload the `CITATION` file which will become a link on the CRAN package. For details on R package citation, please see https://cran.r-project.org/doc/manuals/r-release/R-exts.html#CITATION-files

To sum up, I would prefer to stick to current description for reasons above.

Reply:
Sorry I mainly referred to the content rather than to the syntax (I used markdown above).  OK so please change to:

"""
R implementation of 'PsychroLib'(<https://github.com/psychrometrics/psychrolib>) to calculate psychrometric properties of moist and dry air available in both metric (SI) and imperial (IP) systems of units.
"""